externs/browser/w3c_event.js

```diff
@@ -134,6 +134,12 @@ Event.prototype.timeStamp;
 Event.prototype.path;
 
 /**
+ * Present for events spawned in browsers that support shadow dom.
+ * @type {function():Array<!EventTarget>|undefined}
```

Comment:
Now, it's described in last editor version: http://w3c.github.io/webcomponents/spec/shadow/ . Can I add a link to it or I should wait when it'll become latest published version?